pipelines/curate_bridge_raster.py

```
@@ -0,0 +1,507 @@
+        return value if value >= 0 else None
+    if not isinstance(value, float):
+        return None
+    if not math.isfinite(value):
+        return None
+    if not value.is_integer():
+        return None
+    integer = int(value)
+    return integer if integer >= 0 else None
+
+
+def _positive_int(value: Any) -> bool:
```

Ruff (0.16.2)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `value`

(ANN401)